sdk/template/perf-tests/template/test/index.spec.ts

```diff
@@ -1,11 +1,9 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import { PerfProgram, selectPerfTest } from "@azure/test-utils-perf";
+import { createPerfProgram } from "@azure/test-utils-perf";
```

Comment:
Can you also update the `GettingStarted.md` document and the wiki page for writing perf tests to reflect these?

Reply:
Done